G4_DailyReport/src/main/java/com/example/G4_DailyReport/controller/manager/ProjectProcessController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.G4_DailyReport.controller.manager;
+
+import com.example.G4_DailyReport.model.Project;
+import com.example.G4_DailyReport.model.ProjectProcess;
+import com.example.G4_DailyReport.service.ProjectProcessService;
+import groovy.lang.GString;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@Controller
+@RequestMapping("/manager")
```

Comment:
```suggestion
@RequestMapping("/manager/project-processes")
```
ri để đỡ lặp code dc cu nè